flink-examples-gcp/pom.xml

```diff
@@ -172,6 +174,75 @@ under the License.
       <artifactId>guava</artifactId>
       <version>32.1.3-jre</version>
     </dependency>
+
+    <!-- HBase connector dependencies -->
+
+    <!-- https://mvnrepository.com/artifact/org.apache.flink/flink-connector-hbase-2.2 -->
```

Comment:
@clmccart Do we really need that much dependency ?

Reply:
sorry hadnt pushed the latest snapshot. removed!